pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -0,0 +1,234 @@
+package policy
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
+)
+
+const (
+	AddSCCToGroupRecommendedName      = "add-scc-to-group"
```

Comment:
I can be convinced either way, especially if there is an existing pattern.  I have been looking at it from the SCC side the whole time so I have always considered it 'add user to scc'.  I could definitely see how you could consider the SCC equivalent to a role.  I'm fine with leaving it as is.